extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -3,15 +3,18 @@
 'use strict';
 
 import { Backend, FcUuidAuth } from '../../../../js/common/api/backend.js';
-import { BaseMailFormatter } from './base-mail-formatter.js';
-import { ComposerResetBtnTrigger } from '../compose-err-module.js';
 import { Mime, SendableMsgBody } from '../../../../js/common/core/mime.js';
 import { NewMsgData, PubkeyResult } from '../compose-types.js';
 import { Str, Value } from '../../../../js/common/core/common.js';
+
+import { AcctStore } from '../../../../js/common/platform/store/acct-store.js';
```

Comment:
It appears you have installed one of the import-ordering vscode extensions.

Me and @limonte have found it to be somewhat annoying because it causes syntax highlighting to flash while it's updating, so we dropped it.

If it's still being recommended to add it, we should probably remove that recommendation.

Reply:
Yes, `sort-imports` is in recommended and I agree that we should drop it from that list: #2713